common/hooks/post-install/14-pkglint-locale-in-usr-lib.sh

```diff
@@ -0,0 +1,10 @@
+# vim: set ts=4 sw=4 et:
+#
+# This hook executes the following tasks:
+#   - Looks on all packages for l10n files being installed to /usr/share
+
+hook() {
+    if [ -d $PKGDESTDIR/usr/lib/locale ]; then
+        msg_error "${pkgver}: l10n files found in /usr/lib\n"
```

Comment:
Message would be more helpful with tip how to proceed, eg. _l10n files found in /usr/lib; use /usr/share_

Reply:
Maybe mention `locale` explicitly? So `/usr/lib/locale` and `/usr/share/locale`.

Reply:
just put it in the same hook that checks other banned directories...